vulnerableApp/vulnerableApp_score.py

```diff
@@ -0,0 +1,481 @@
+# Zed Attack Proxy (ZAP) and its related class files.
+#
+# ZAP is an HTTP/HTTPS proxy for assessing web application security.
+#
+# Copyright 2012 ZAP Development Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script tests ZAP against Owasp VulnerableApp: https://github.com/SasanLabs/VulnerableApp
+# Note: Owasp VulnerableApp has to be installed somewhere - the above link is to the github
+# project not the test suite!
+#
+# To use this script:
+# * Install the ZAP Python API:
```

Comment:
faced issue because of Python2 in mac by default. do we need to consider adding the same in readme.md ?